src/tests/test_qir.py

```diff
@@ -242,3 +243,25 @@ def test_qir_out_of_order_measure_declaration(self):
             _get_qir_path("out_of_order_measure.ll"), compiler_config=config
         )
         assert len(results) == 4
+
+    @pytest.mark.parametrize(
+        "qir_file",
+        [
+            "base_profile_ops.ll",
+            "basic_cudaq.ll",
+            "bell_psi_minus.ll",
+            "bell_psi_plus.ll",
+            "bell_theta_minus.ll",
+            "bell_theta_plus.ll",
+            "complicated.ll",
+            "generator-bell.ll",
+            "hello.bc",
+            "out_of_order_measure.ll",
+            "select.bc",
+        ],
+    )
+    def test_on_jagged_hardware(self, qir_file):
+        hw = get_default_echo_hardware(8)
```

Comment:
Meant to be `get_jagged_echo_hardware`?

Reply:
Ah, yes. Was confirming that they were all qir files that "should" pass on simple echo hw and forgot to switch back to jagged.